user/forms.py

```diff
@@ -6,36 +9,57 @@
 
 class CustomUserCreationForm(UserCreationForm):
     """
-    This class represents a form the create a new user.
+    A form to create a new user.
     """
     class Meta(UserCreationForm.Meta):
+        """
+        Metadata for the model User. Selects which fields should be editable in this form.
+        """
         model = User
         fields = UserCreationForm.Meta.fields + ('email', 'first_name', 'last_name', 'organisation', 'type_of_use',)
 
 
 class CustomUserChangeForm(UserChangeForm):
     """
-    This class represents a form to change attributes of a user. The Form is used in the admin area.
+    A form to change attributes of a user. This form is only used in the admin area.
     """
     class Meta(UserChangeForm.Meta):
+        """
+        Metadata for the model User. Selects which fields should be editable in this form.
+        """
         model = User
         fields = '__all__'
 
 
 class CustomUserEditForm(UserChangeForm):
     """
-    This class represents a form to change attributes of a user by himself.
+    A form to change attributes of a user by himself.
+
+    Attributes
+    ----------
+    password : str
+        None - to erase the password in the form.
     """
     password = None
 
     class Meta(UserChangeForm.Meta):
+        """
+        Metadata for the model User. Selects which fields should be editable in this form.
+        """
         model = User
         fields = ('username', 'first_name', 'last_name', 'email', 'organisation', 'type_of_use')
 
 
 class CustomUserAdmin(UserAdmin):
     """
-    This class provides which fields are seen in the admin area.
+    Provides which fields are seen in the admin area.
```

Comment:
Provides the X? If it is not clear to whom the CUstomUserAdmin provides these fields, maybe "defines" is a better word?